context.go

```diff
@@ -462,13 +457,7 @@ func (c *Context) GetPostFormMap(key string) (map[string]string, bool) {
 			debugPrint("error on parse multipart form map: %v", err)
 		}
 	}
-	dicts, exist := c.get(req.PostForm, key)
-
-	if !exist && req.MultipartForm != nil && req.MultipartForm.File != nil {
-		dicts, exist = c.get(req.MultipartForm.Value, key)
-	}
```

Comment:
same as above.

Reply:
Add this code for support go1.6, please commit message: https://github.com/gin-gonic/gin/pull/1383/commits/51386de165ae50a61c06d80f06ec703810a2d62d